backend/templates/cms_index.html

```diff
@@ -0,0 +1,158 @@
+                    {% else %}
+                          <!-- Other blog entries-->
+                            <div class="card mb-4 border-0" style="background-color: #70D8F112;max-height:227px ">
+                                <div class="card-body" style="max-width: 696px;text-overflow: ellipsis;  overflow: hidden;">
+                                    <a href="#" class="card-title h3 mb-auto" style="text-decoration: none;
+                                    color:#FB2F2A;max-height: 25px;overflow:hidden;text-overflow: ellipsis">
+                                        {{ post.title|truncatewords:15 }}</a>
+                                    <h6 class="card-subtitle mb-1 mt-1 text-muted text-uppercase">{{ post.profession }}</h6>
+                                    <h5 style="color: #3295B8">{{ post.author }} <small class="text-muted
+                                    text-black">{{ post.published }}</small>
+                                    </h5>
+                                    <div class="list-inline mb-4 mt-3">
+                                        {% for genre in post.category.get_queryset %}
+                                            <a class="btn btn-outline-danger btn-sm mb-auto text-black" href="#"
+                                               role="button">{{ genre.tag}}</a>
+                                        {% endfor %}
+                                    </div>
+                                </div>
+                            </div>
+                    {% endif %}
+                {% endfor %}
+            </div>
+            <div class="col-lg-4">
+               <!-- Tags widget-->
+                <div class="card mb-4 border-0">
+                    <div class="card-body">
+                        <div class="card-title mb-1 fw-bold">Tags</div>
+                        <div class="container">
+                            {% for tag in tags %}
+                                <a class="btn btn-outline-danger btn-sm mr-md-1 my-1 text-black" href="#"
+                                   role="button">{{ tag.tag }}</a>
+                            {% endfor %}
+                        </div>
+                    </div>
+                </div>
+                 <!-- Post Links-->
+                 <div class="card mb-4 border-0">
+                    <div class="card-body">
+                        <div class="title mb-1 fw-bold">Posts</div>
+                        <ul class="list-unstyled mb-0">
+                            {% for post in posts %}
+                            <li style="max-width: 479px"><a class="post-link"  href="#" style="color: #3295B8;">
+                                {{ post.title|truncatewords:4 }}
+                            </a> {{ post.published }}</li>
+                            {% endfor %}
+                        </ul>
+                    </div>
+                </div>
+            </div>
+
+         </div>
+
+
+
+    </div>
+
+{% endblock %}
+
+
+
+
+
+
+
```

Comment:
Be mindful of the whitespace you're adding. Code should be orderly — it's harder to know what's going on with all the blank lines